pubsublite-spark-sql-streaming/src/main/java/com/google/cloud/pubsublite/spark/PslDataSourceOptions.java

```diff
@@ -32,34 +32,23 @@
 public abstract class PslDataSourceOptions implements Serializable {
   private static final long serialVersionUID = 2680059304693561607L;
 
-  @Nullable
-  public abstract String credentialsAccessToken();
-
   @Nullable
   public abstract String credentialsKey();
 
-  @Nullable
-  public abstract String credentialsFile();
-
   public abstract SubscriptionPath subscriptionPath();
 
   public abstract long maxBytesOutstanding();
 
   public abstract long maxMessagesOutstanding();
```

Comment:
nit. If users never set this, can you make it a FlowControlSettings?

Reply:
Done